javaslang/src-gen/main/java/javaslang/API.java

```diff
@@ -84,6 +90,573 @@ private API() {
     }
 
     //
+    // Aliases for static factories
+    //
+
+    /**
+     * Alias for {@code Function0.of()}
```

Comment:
We need to generate valid Javadoc (or leave it completely away but that is no viable option for us) - otherwise `mvn javadoc:javadoc` will fail. We need that maven target for a release, so please test it (ideally on every PR).

I suggest Javadoc of the following form. Using `@link` instead of `@code` in the first line might come handy when adding more aliases.

```java
/**
 * Alias for {@link Function0#of(Function0)}.
 *
 * @param methodReference A method reference
 * @return A {@code Function0}
 */
public static <R> Function0<R> f0(Function0<R> methodReference) {
    return Function0.of(methodReference);
}
```